src/PackageVersions/Installer.php

```diff
@@ -176,4 +178,14 @@ private static function getVersions(Locker $locker, RootPackageInterface $rootPa
 
         yield $rootPackage->getName() => $rootPackage->getVersion() . '@' . $rootPackage->getSourceReference();
     }
+
+    /**
+     * @return array
+     */
+    public function getCapabilities() : array
+    {
+        return [
+            CommandProviderInterface::class => CommandProvider::class,
```

Comment:
Unfortunately, there's no easy way to add a command to a Composer plugin, you have to do all this CommandProvider stuff :(